core/storage/user/gae_models.py

```diff
@@ -2367,7 +2367,8 @@ class PendingDeletionRequestModel(base_models.BaseModel):
     # IDs of all the private collections created by this user.
     collection_ids = ndb.StringProperty(repeated=True, indexed=True)
 
-    # Story to pseudonymous user ID mapping.
+    # Dict of story IDs as keys and pseudonymous user IDs as values. For
+    # different story we used different pseudonymous user ID.
```

Comment:
A dict mapping story IDs to pseudonymous user IDs. For each story, we use a different pseudonymous user ID. Note that all these pseudonymous user IDs originate from the same about-to-be-deleted user.

Reply:
Done.